Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -131,6 +140,17 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
         this.setState({ ...this.state, quantity }, this.validate);
     }
 
+    updateNewClientInfo = (info: INewClientInfo) => {
+        this.setState({...this.state,
+            newClientInfo: {

```

Comment:
@laholstege if you wanted to here's another change to make this a one-liner using the spread operator since the info param passed in is of the same type as your state.